src/components/lib.js

```diff
@@ -0,0 +1,8 @@
+import CyverseAnnouncer, { announce } from "./announcer/CyverseAnnouncer";
+import Rate from "./rating/Rate";
+
+export {
+    CyverseAnnouncer,
+    announce,
+    Rate,
```

Comment:
They way we have the `build` script and this export set up now, I think we would have to use something like the following in our main UI code to use these components:
```javascript
import { CyVerseAnnouncer, announce, Rate } from "@cyverse-de/de-components";
```

I wonder if we should imitate a package structure similar to [material-ui](https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/index.js) (and we should decide soon before these components are imported in too many places)?

For example, to use a MUI `Button` and an icon:
```javascript
import Button from "@material-ui/core/Button";
import HelpIcon from "@material-ui/icons/Help";
```

Would it be more convenient to use imports like the following for our components?
```javascript
import CyVerseAnnouncer from "@cyverse-de/de-components/CyVerseAnnouncer";
import Rate from "@cyverse-de/de-components/Rate";
import SomethingSoon from "@cyverse-de/de-components/future-sub-package/SomethingSoon";
```

Basically treating `@cyverse-de/de-components` the same as `@material-ui/core`, but sub-packaging from there rather than creating separate packages (like `@material-ui/icons`).

I don't know the best way to organize the exports to accomplish this right now, but we could experiment and figure something out if we wanted to go this way.

Reply:
I think it'd be more convenient to have the imports like how MUI has them, but also not sure how much work it would take to replicate MUI's package structure.